presto-main/src/test/java/com/facebook/presto/operator/index/TestTupleFilterProcessor.java

```diff
@@ -51,47 +49,13 @@ public void testFullPageFilter()
                 .row("a", 0L, false, 0.2, 0.2)
                 .build());
 
-        PageBuilder pageBuilder = new PageBuilder(outputTypes);
-        int end = tupleFilterProcessor.process(SESSION, inputPage, 0, inputPage.getPositionCount(), pageBuilder);
-        Page actualPage = pageBuilder.build();
+        Page actualPage = tupleFilterProcessor.process(SESSION, inputPage, outputTypes);
```

Comment:
Rename test method as it is misleading provided we no longer support processing part of the page